hack/make/manifests/kubernetes.mk

```
@@ -2,8 +2,13 @@
 manifests/kubernetes/csi:
 	# Generate kubernetes-csi.yaml
 	helm template dynatrace-operator config/helm/chart/default \
+		-s templates/Common/csi/clusterrole-csi.yaml \
+		-s templates/Common/csi/csidriver.yaml \
+		-s templates/Common/csi/daemonset.yaml \
+		-s templates/Common/csi/priority-class.yaml \
+		-s templates/Common/csi/role-csi.yaml \
+		-s templates/Common/csi/serviceaccount-csi.yaml \
```

it seems the `--show-only` parameter supports regex, so we could just simplify this (and make it harder to forget to add new files) by adding all files in the `templates/common/csi` folder.

```suggestion
		-s templates/Common/csi/*.yaml \
```

---

done

---

I don't see this change in the latest commit, you only changed `-s` to `--show-only`

---

sorry, can't read 🙈 

also you fixed my sidenote: 😘  ❤️  
> Sidenote: I tried to make it "prettier" by trying not to list each CSI-related file 1-by-1, but didn't find a way (it's not pretty, but still much prettier than partial)